.travis.yml

```diff
@@ -16,6 +16,15 @@ before_install:
   rm -f install.sh
 install:
 - go get github.com/golang/lint/golint
+# Install protobuf dependencies
+- go get -u github.com/golang/protobuf/protoc-gen-go
+- curl -OL https://github.com/google/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip
+- sudo unzip -o protoc-3.7.1-linux-x86_64.zip -d /usr/local bin/protoc
+- sudo unzip -o protoc-3.7.1-linux-x86_64.zip -d /usr/local include/*
+- rm -f protoc-3.7.1-linux-x86_64.zip
```

Comment:
Is this necessary? If we keep installing it every time, that implies the Travis storage space is temporary.

Reply:
From what I understood, Travis spawns a new VM on every commit/PR. Besides setting up the languages (as we do above with 2 version of `Golang`), there's no hard "VM dependencies" that we can have only one time.

read: I do not know how to do it otherwise

Reply:
I think Marcial is correct - when we added bazel builds to Travis we had do re-download and install bazel on every presubmit run like this.